interfaces/portal/src/app/services/registrations-table-column.service.ts

```diff
@@ -243,6 +112,211 @@ export class RegistrationsTableColumnService {
     });
   }
 
+  private createBasicColumns(
+    projectId: Signal<number | string>,
+    project: Project,
```

Comment:
Could the parameter `projectId` not be taken out of `project`? i.e. `project.id`?

It would make all the "create * Columns"-methods look even more alike. ;)

Reply:
Duh, good point. Changed it.